crates/sui-graphql-rpc/src/types/digest.rs

```diff
@@ -56,6 +60,12 @@ impl TryFrom<&[u8]> for Digest {
     }
 }
 
+impl From<Digest> for ObjectDigest {
+    fn from(digest: Digest) -> Self {
+        ObjectDigest::new(digest.into_inner())
```

Comment:
nit (optional): You might a well just inline the impl for `into_inner` here -- I think this is the only place you use it.

```suggestion
        ObjectDigest::new(digest.0)
```